setup.py

```diff
@@ -50,7 +50,7 @@
 ]
 
 install_requires = [
-    "reana-commons[snakemake_reports]>=0.9.4,<0.10.0",
+    "reana-commons[snakemake_reports]>=0.9.5,<0.10.0",
```

Comment:
This will need to be changed to 0.9.6 after we release the new version of r-commons